products/zoningtaxlots/models/intermediate/int__limitedheight.sql

```diff
@@ -33,18 +33,19 @@ limitedheightper AS (
 limitedheightperorder AS (
```

Comment:
Wow this one had a lot of unused stuff happening.

Looks like this can be simplified a bit further - the two separate CTEs here aren't really doing anything distinct, so we can merge them.

Also, we can simplify the result. The CASE statement is making the value to be joined to our final table either NULL or the appropriate value. But the output of this model is meant to be a table of pairs that gets joined, so NULLs aren't really needed. Meaning the boolean value in the CASE statement can really just be in a WHERE clause instead to filter the rows in the model